setup_dev.sh

```diff
@@ -31,7 +31,9 @@ sed -i "s@^  :password.*@  :password => '$mysql_congress_forms',@" config/databa
 
 HOME=/home/vagrant sudo -u vagrant /bin/bash <<EOF
 echo "Setting up RVM and Ruby..."
-curl -sSL https://get.rvm.io | bash -s stable
+gpg --keyserver hkp://keys.gnupg.net --recv-keys D39DC0E3
+command curl -sSL https://rvm.io/mpapis.asc | gpg --import -
```

Comment:
Hm I'm running into this issue where rvm isn't getting installed correctly
![screen shot 2015-02-02 at 5 56 36 pm](https://cloud.githubusercontent.com/assets/1928424/6010950/dd48d474-ab04-11e4-8bf5-2beff721ad2b.png)